src/js/jsx/sections/style/ShadowList.jsx

```diff
@@ -219,7 +219,7 @@ define(function (require, exports) {
                     blendMode,
                     this.props.type);
             // Currently does not differentiate between inner and drop shadows
-            headlights.logEvent("edit", "shadow-blendmode-input", blendMode);
+            headlights.logEventDebounced("edit", "shadow-blendmode-input", blendMode);
```

Comment:
I agree. Ideally we should log the event based on the last selection in DataList, but that seems risky to make the change at the last moment, compare to using debounced logEvent. We may consider lowering the delay time from 1500 to 1000.